packages/workspace/src/generators/ci-workflow/files/azure/azure-pipelines.yml__tmpl__

```diff
@@ -55,8 +55,9 @@ jobs:
       # This enables task distribution via Nx Cloud
       # Run this command as early as possible, before dependencies are installed
       # Learn more at https://nx.dev/ci/reference/nx-cloud-cli#npx-nxcloud-startcirun
-      <% if (connectedToCloud) { %>- script: <%= packageManagerPreInstallPrefix %> nx-cloud start-ci-run --distribute-on="3 linux-medium-js" --stop-agents-after="<% if(hasE2E){ %>e2e-ci<% } else { %>build<% } %>"
-      <% } else { %># Connect your workspace by running "nx connect" and uncomment this
+      <% if (connectedToCloud) { %># Uncomment this line to enable task distribution
```

Comment:
We can remove the if-else since it's always commented out now. The instruction to uncomment to enable distribution should be the same.

Same thing for the other CI templates.

Reply:
The if else is checking if the workspace is connected to Nx cloud already.  Not whether they want task distribution